File: app/src/main/java/ru/nikshlykov/englishwordsapp/db/setting/Setting.java
package ru.nikshlykov.englishwordsapp.db.setting;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "Settings")
public class Setting {

    @PrimaryKey()
    @NonNull
    @ColumnInfo(name = "_id")
    public long id;

    @NonNull
    @ColumnInfo(name = "Param")
    public String name; // Имя параметра для вывода в Activity настроек.

    @NonNull
    @ColumnInfo(name = "Value", defaultValue = "0")
    public int value; // Значение параметра настроек.

}

File: app/src/main/java/ru/nikshlykov/englishwordsapp/db/setting/Setting.kt
package ru.nikshlykov.englishwordsapp.db.setting

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Settings")
class Setting {
  @PrimaryKey
  @ColumnInfo(name = "_id")
  var id: Long = 0

  @ColumnInfo(name = "Param")
  var name: String = "" // Имя параметра для вывода в Activity настроек.

  @ColumnInfo(name = "Value", defaultValue = "0")
  var value = 0 // Значение параметра настроек.
}